src/components/stopwatch/StopWatchController.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <FreeRTOS.h>
+#include <optional>
+#include <timers.h>
+#include "utility/CircularBuffer.h"
+
+namespace Pinetime {
+  namespace System {
+    class SystemTask;
+  }
+  namespace Controllers {
+
+    enum class StopWatchStates { Cleared, Running, Paused };
+
+    struct LapInfo {
+      int count = 0;       // Used to label the lap
```

Comment:
I think the default values can be removed as they're always set in `Clear()`

Reply:
Initializers in declarations provide additional information in a convenient place, so I'm usually OK with them as long as they're not too expensive. Is there a particular reason in C++ or in InfiniTime to prefer imperative intialization? I know we need a `Clear()` method anyway, so are we going for code size savings here?

Reply:
Applied.

Reply:
On reflection you make a good point, there's no harm in default values and the performance/space cost is very small (if any)